Migrate the audit charm to sidecar (#7)
charmed-kubernetes/opa-gatekeeper-operators: opa-audit-operator/README.md

# Open Policy Agent Gatekeeper Audit Operator
## Description

This is the OPA Gatekeeper audit operator charm.
[OPA gatekeeper](https://open-policy-agent.github.io/gatekeeper/website/docs/)
is an open source, general-purpose policy engine that enables unified,
context-aware policy enforcement.

> The `opa-manager-audit` charm is used to apply policies to incomming

Reviewer: This should probably say something like "The `opa-manager-audit` charm is used to audit existing resources blah blah blah"

kubernetes API requests.

## Usage
### Metrics

> Gatekeeper Controller Manager metrics can be integrated with a deployed

Reviewer: ```suggestion
Gatekeeper Audit metrics can be integrated with a deployed
```

[prometheus-k8s operator](https://charmhub.io/prometheus-k8s) using the following command:
```commandline

> $ juju relate gatekeeper-controller-manager prometheus-k8s

Reviewer: ```suggestion
$ juju relate gatekeeper-audit prometheus-k8s
```

```

If you would like to rely on the [grafana-agent-k8s operator](https://charmhub.io/grafana-agent-k8s) to push metrics,
you can use the commands below:
```commandline

> $ juju relate grafana-agent-k8s gatekeeper-controller-manager

Reviewer: ```suggestion
$ juju relate grafana-agent-k8s gatekeeper-audit
```
